docs/docs/manual/exploring.md

```diff
@@ -1,102 +1,346 @@
+*   boolean (values of “true” or “false”)
+*   date (ISO-8601-compliant extended format with time in UTC: **YYYY**-**MM**-**DD**T**HH**:**MM**:**SS**Z)
+*   array (a set of comma-separated string, number, boolean, or date values)
+*   error
+*   null
 
-### Null vs empty string
+An “error” data type is created when the cell is storing an error generated during a transformation in OpenRefine.
 
+A “null” data type is a special value which basically means “this cell has no value.” It’s used to differentiate between cells that have values such as “0” or “false” - or a cell that looks empty but has, for example, spaces in it. When you use `type(value)`, it will show you that the cell’s value is “null” and its type is “undefined.” 
 
+You can opt to show “null” values to differentiate them from empty strings, by going to “All” > “View” > “Show/Hide ‘null’ values in cells.”
 
-*   Viewing null values
+To transform data from one type to another, see [Transforming data](transforming-data#transform) for information on using `toString()`, `toDate()`, and other functions.
 
+## Row types: rows vs. records
 
-### Rows vs. records
+A row is a simple way to organize data: a series of cells, one cell per column. Sometimes there are multiple pieces of information in one cell, such as when a survey respondent can select more than one response. In cases where there is more than one value for a single column in one or more rows, you may wish to use OpenRefine’s records mode: this defines a single record (a survey response, for example) as potentially containing more than one row. From there you can transform cells into multiple rows, each cell containing one value you’d like to work with. 
 
+Generally, when you import some data, OpenRefine reads that data in row mode. From there you can convert the project into records mode. OpenRefine remembers this action and will present you with records mode each time you open the project from then on. 
 
+OpenRefine understands records based on the content of the first column. Splitting a row into a multi-row record will base all association on the first column in your dataset. If you have more than one column to split out into multiple rows, OpenRefine will keep your data associated with its original record: you can imagine this structure as a tree with many branches, all leading back to the same trunk. 
 
-*   Functions that matter to rows/records distinction
+OpenRefine assigns a unique key behind the scenes, so your records don’t need a unique identifier. You can keep track of which rows are assigned to which record by the record number that appears under the “All” column.
 
+To [split multi-valued cells](transforming-data#split-multi-valued-cells) and apply other operations that take advantage of records mode, see [Transforming data](transforming-data). 
+
+:::caution
+Be careful when in records mode that you do not accidentally delete rows based on being blank in one column where there is a value in another. 
+:::
```

Comment:
It's not clear to me that this is the single most salient sort of issue we should flag about the records mode - perhaps it could be left in normal text without the caution box (which really jumps to the eyes).